images/onnx-predictor-cpu/Dockerfile

```diff
@@ -50,6 +49,14 @@ RUN pip install --no-cache-dir -r \
     /src/cortex/serve/requirements.txt \
     -r /src/cortex/serve/image.requirements.txt
 
+COPY pkg/workloads/cortex/serve/requirements.txt /src/cortex/serve/requirements.txt
+COPY pkg/workloads/cortex/serve/onnx-cpu.requirements.txt /src/cortex/serve/image.requirements.txt
+COPY pkg/workloads/cortex/internal/requirements.txt /src/cortex/internal/requirements.txt
+RUN pip install --no-cache-dir -r \
+    /src/cortex/serve/requirements.txt -r \
+    /src/cortex/serve/image.requirements.txt -r \
```

Comment:
what is your motivation for leaving the serve folder, if everything else is moved into internal?

Reply:
There are initialization scripts (in bash), there's the bootloader, there's the readiness checker, there are the uvicorn services, all of which fall outside the scope of a Python package. The internal Python package called `cortex_internal` contains the building blocks for the webserver, whereas the initialization scripts just glues them together. And these have different dependencies, for which there are different packages lists of dependencies.

This is akin to having a script that uses `numpy` as a library, where `numpy` would be the `cortex_internal` package. It's supposed to act as a library.

Reply:
From my point of view it looks like you are separating files that are related to the same web application. The shell scripts and configuration, in the end are part of the `serve.py` web server entrypoint. 

To me it does not make much sense to separate the entrypoint from the app. Everything should be under the same folder, to promote code readability.

I would suggest one of two options:

1) Move everything back to `serve` from `internal` and just rename the previously named `cortex` module to `cortex_internal` as its named now.

2) Move the contents of `serve` to `internal`. 

This would avoid having 2 separate `requirements.txt` files and would overall simplify the overview of the code structure from a new user point of view.

It would also avoid having the code under `serve` referring to code under `internal`

e.g. https://github.com/cortexlabs/cortex/blob/eb1ff1dfd0b2f8cce02fd1a72342422344f385be/pkg/workloads/cortex/serve/start/server.py#L32-L39

Reply:
As we have discussed, option one has been applied to this PR.